optuna/trial/_trial.py

```diff
@@ -384,14 +388,23 @@ def objective(trial):
                 Lower endpoint of the range of suggested values. ``low`` is included in the range.
             high:
                 Upper endpoint of the range of suggested values. ``high`` is included in the range.
+            log:
+                A flag to sample the value from the log domain or not.
+                If ``log`` is true, the value is sampled from the range in the log domain.
+                Otherwise, the value is sampled from the range in the linear domain.
```

Comment:
As I mentioned in https://github.com/optuna/optuna/pull/1201/files#r426442884, we need more explanation about the specification of `trial.suggest_int(... log=True ...)`. Could you write the documentation in more detail? (How the domain space `[low, high]` is log converted. How the domain space is divided by `step`. How the distribution of sampled points looks like. How the set of possible sampled points looks like.) 

If it needs a lot of work, I will do it in the separate PR! :)

Reply:
Thank you for your comment! I update the document, please see https://github.com/optuna/optuna/pull/1201/commits/73b5033bba9bbadf5b629d876da4225696f40c89#diff-ba717fa8dc7e3ae6816dd57373267dc4R393-R402 



How do you think?

Reply:
Thanks! I put some minor suggestions below!